Makefile

```diff
@@ -0,0 +1,10 @@
+
+check_if_gh_is_installed:
+	@if ! command -v gh &> /dev/null; then\
+		echo "gh is not installed";\
+		exit 1;\
+	fi
```

Comment:
I would prefer this moved to the script, to keep the make file clean. As we add more validation (e.g. for `GITHUB_TOKEN`, as I commented before) it's better to keep it centralized.

Reply:
Moved and done!